css/emailverification.css

```diff
@@ -0,0 +1,41 @@
+
+body{
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+  background: rgba(196, 196, 196, 0.17);  
+}
+
+.display{
+  height: 350px;
+  background: #1F59BB;
+}
+
+.emailVer-container{
+  margin-top: 3rem;
+  height: 800px;
+  padding: 30px;
+  background: #FFFFFF;
+  color: #1F59BB;
+  box-shadow: 0px 1px 3px rgba(53, 151, 245, 0.2);
+}
+
+.emailVer-text h1{
+   font-size: 3.5rem;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Expected indentation of 2 spaces (indentation)](https://app.codacy.com/gh/codeclannigeria/CodeClanUI/pullRequest?prid=5674265)